client/src/app/learn/learn.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-learn',
+  templateUrl: './learn.component.html',
+  styleUrls: ['./learn.component.scss']
+})
+export class LearnComponent implements OnInit {
+  wordList: any[] = [
+    {
+      'en': 'Dance',
+      'pl': 'Tańczyć'
+    },
+    {
+      'en': 'Talk',
+      'pl': 'Mówić'
+    },
+    {
+      'en': 'Walk',
+      'pl': 'Chodzić'
+    },
+  ];
+  currentId: number = 0;
+  lang: string = 'en';
+  secondLang: string = 'pl';
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
+  toggleLanguage(): void {
```

Comment:
jeśli to ma być toggle Language, to zastanowiłbym się nad zrobieniem flagi boolean
this.lang = !this.lang